### PR TITLE
Migrate BugBountyCards component to chakra UI [Relates to #6374]

### PR DESCRIPTION
**Pull request by Marcelixoo**
## Description

This PR implements the BugBountyCards component using ChakraUI.

Below you can find screenshots of the component rendered before/after the changes side-by-side and in different states.

### Dark mode: Large screens
1. Current versions
Screenshot 2022-11-25 at 19 58 01
ion

2. After changes
Screenshot 2022-11-25 at 19 58 07

### Light mode: Large screens
1. Current version
Screenshot 2022-11-25 at 19 58 28

2. After changes
Screenshot 2022-11-25 at 19 58 19

### Light mode: Medium screens
1. Current version
Screenshot 2022-11-25 at 19 58 45

2. After changes
Screenshot 2022-11-25 at 19 58 50

### Light mode: Small screens
1. Current version
Screenshot 2022-11-25 at 19 59 06

2. After changes
Screenshot 2022-11-25 at 19 59 14

## Related Issue
#6374

### CODE REVIEW

**Comment by Marcelixoo** [comment]
FYI, I've checked the deployment preview and it looks pretty solid there as well 👍🏼

File: src/components/BugBountyCards.tsx [hunk 1]
```diff
@@ -1,113 +1,160 @@
...
+      justifyContent="space-between"
+      bg="background"
+      borderRadius="2px"
+      boxShadow="var(--eth-colors-tableBoxShadow)"
+      border="1px solid"
+      borderColor="border"
+      m={4}
+      _hover={{
+        "border-radius": "4px",
+        "box-shadow": "var(--eth-shadows-tableBoxHover)",
```
Comment (pettinarip) [inline src/components/BugBountyCards.tsx:47]:
```suggestion
        boxShadow: "tableBoxHover",
```

**Review by pettinarip — CHANGES_REQUESTED** [review]
Looks great overall, nice job @Marcelixoo. I've left a few minor comments.

File: src/components/BugBountyCards.tsx [hunk 2]
```diff
@@ -1,113 +1,160 @@
...
+
+const Card = ({ children, ...props }: FlexProps) => {
+  return (
+    <Flex
+      flexDir="column"
+      flex={{ base: "1 1 412px", xl: "1 1 260px" }}
+      justifyContent="space-between"
+      bg="background"
+      borderRadius="2px"
+      boxShadow="var(--eth-colors-tableBoxShadow)"
```
[the remark this answers is not in the document]
Comment (Marcelixoo) [reply src/components/BugBountyCards.tsx]: Thanks, @pettinarip! I think I've addressed all comments on this thread.

Could you give it a second look?

**Review by pettinarip — APPROVED** [review]
Nicee! @Marcelixoo thanks. LGTM 🚀

**Comment by pettinarip** [comment]
Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by pettinarip** [comment]
@all-contributors please add @Marcelixoo for code

**Pull request merged**